scripts/compute_csa.sh

```diff
@@ -0,0 +1,237 @@
+#!/bin/bash
+#
+# Compute the CSA of a lifelong learning contrast-agnostic segmentation model on the spine-generic test dataset.
+# To be used within the `compute_morphometrics_spine_generic.sh` script to run the CSA analysis on the latest version 
+# of the contrast-agnostic model
+#
+# Author: Naga Karthik
+#
+
+# Uncomment for full verbose
+set -x
+
+# Immediately exit if error
+set -e -o pipefail
+
+# Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
+trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
+
+# Print retrieved variables from the sct_run_batch script to the log (to allow easier debug)
+echo "Retrieved variables from from the caller sct_run_batch:"
+echo "PATH_DATA: ${PATH_DATA}"
+echo "PATH_DATA_PROCESSED: ${PATH_DATA_PROCESSED}"
+echo "PATH_RESULTS: ${PATH_RESULTS}"
+echo "PATH_LOG: ${PATH_LOG}"
+echo "PATH_QC: ${PATH_QC}"
+
+SUBJECT=$1
+CUDA_DEVICE=$2
+PATH_NNUNET_SCRIPT=$3   # path to the nnUNet contrast-agnostic run_inference_single_subject.py
+PATH_NNUNET_MODEL=$4
+
+echo "SUBJECT: ${SUBJECT}"
+echo "USING CUDA DEVICE ID: ${CUDA_DEVICE}"
+echo "PATH_NNUNET_SCRIPT: ${PATH_NNUNET_SCRIPT}"
+echo "PATH_NNUNET_MODEL: ${PATH_NNUNET_MODEL}"
+
+# ------------------------------------------------------------------------------
+# CONVENIENCE FUNCTIONS
+# ------------------------------------------------------------------------------
+
+# Check if manual label already exists. If it does, copy it locally.
+# NOTE: manual disc labels should go from C1-C2 to C7-T1.
+label_vertebrae(){
+  local file="$1"
+  local contrast="$2"
+
+  # Update global variable with segmentation file name
+  FILESEG="${file}_softseg_bin"
+  FILELABEL="${file}_discs"
+
+  # Label vertebral levels
```

Comment:
```suggestion
  # Label vertebral levels
  # Note: we are using sct_label_utils over sct_label_vertebrae here to avoid straightening (which takes a lot of time)
```